libr/bin/obj.c

```diff
@@ -8,17 +8,135 @@
 	if (binfile->rbin->verbose) \
 	eprintf
 
+static void mem_free(void *data) {
+	RBinMem *mem = (RBinMem *)data;
+	if (mem && mem->mirrors) {
+		mem->mirrors->free = mem_free;
+		r_list_free (mem->mirrors);
+		mem->mirrors = NULL;
+	}
+	free (mem);
+}
+
+static void object_delete_items(RBinObject *o) {
```

Comment:
This is called only once from objfree. What about moving all this code in there and remove the function?

Reply:
I'm actually ok having a separate function. I tink it makes things simpler to read.